README.md

```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
 
-    $ polymer install --variants
+Lint your project for common errors. Provide a set of linting rules via either your `polymer.json` configuration file or the `--rules` command option.
 
-This performs a Bower install, and also installs any dependency variants
-specified in the `"variants"` property of `bower.json`. See
-[Dependency Variants](#dependency-variants) above.
-
-### lint
-
-Add a "lint" section to your `polymer.json` file at the root of your project:
+For example, if you wanted to lint a Polymer 2.0 Hybrid element, you would run `polymer lint --rules polymer-2-hybrid`. To make sure the correct rule set is always used, you should add the following field to your `polymer.json`:
 
 ```json
-{
-  "lint": {
-    "rules": [
-      "polymer-2-hybrid"
-    ]
-  }
-}
+"lint": {
+  "rules": [
+    "polymer-2-hybrid"
+  ]
+},
 ```
 
-Then just
-
-    $ polymer lint
-
-You can also list specific files to lint and rules to use on the command line:
-
-    $ polymer lint --rules polymer-2-hybrid --input ./my-element.html
-
-### test
-
-Run test with web-component-tester:
-
-    $ polymer test
+Run `polymer help lint` for the full list of available options.
 
-### build
 
-Specify project files as flags:
+### `polymer test [options...]`
 
-    $ polymer build --entrypoint index.html --shell src/my-app/my-app.html
+Run your element or application tests with [`web-component-tester`](https://github.com/Polymer/web-component-tester/).
 
-Use `index.html` as the entrypoint, or read from `polymer.json`:
+Run `polymer help test` for the full list of available options.
 
-    $ polymer build
 
-`build` is opinionated and defaults to a good build for app-shell apps. It writes the built output to `build/bundled` and `build/unbundled` folders. Both outputs have been run though HTML, JS and CSS optimizers, and have a Service Worker generated for them. The bundled folder contains the application files process by Vulcanize, Polymer's HTML bundler, for optimal loading via HTTP/1. The unbundled folder is optimized for HTTP/2 + Push.
+### `polymer build [options...]`
 
-While the build command should support most projects, some users will need greater control over their build pipeline. If that's you, check out the [polymer-build](https://github.com/Polymer/polymer-build) library. Polymer-build can be called and customized programmatically, giving you much greater control than the CLI can provide. Visit the repo for usage information and examples.
+Build a Polymer application for production. This includes support for site optimizations like code bundling, minification, and ES6 compilation to run on older browsers.
 
-#### --js-compile
+Most optimizations and production enhancements are disabled by default. To make sure the correct build enhancements are always used, you can provide a set of build configurations via the ["builds"](https://www.polymer-project.org/2.0/docs/tools/polymer-json#builds) field of your `polymer.json` file:
```

Comment:
Maybe nix the `production enhancements` part, as it's well covered by `optimizations`